client/README.md

```diff
@@ -1,70 +1,16 @@
-# Getting Started with Create React App
+# Traveler's Tea
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Running the app
 
-## Available Scripts
+`yarn install` to install any dependencies
```

Comment:
@Graywing13 could you please update this to ```cd client``` as well then do a ```yarn install```? As ```yarn start``` didn't work for me in the client by just just doing a ```yarn install``` on the project root directory (could be because we have 2 ```package.json``` files, one in client end and other on server)

Reply:
yes, there is no package.json in root, so you can't do yarn install in root. it is probably better to just move line 9 up

Reply:
Right I could